app/istqb-ctel-tm-mtt-complete-guide.css

```
@@ -0,0 +1,1130 @@
+            body::after {
+                content: '';
+                position: fixed;
+                inset: 0;
+                pointer-events: none;
+                z-index: 0;
+                background-image:
+                    linear-gradient(rgba(0, 212, 255, 0.04) 1px, transparent 1px),
+                    linear-gradient(90deg, rgba(0, 212, 255, 0.04) 1px, transparent 1px);
+                background-size: 40px 40px;
+            }
+            @keyframes scanMove {
```

Stylelint (17.12.0)

[error] Expected keyframe name "scanMove" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)